files/usr/lib/lua/aredn/utils.lua

```diff
@@ -257,6 +257,20 @@ function nslookup(ip)
         end
 end
 
+-------------------------------------
+-- Returns first IP of given host
+-------------------------------------
+function iplookup(host)
+	local ip=nil
+	local nso=nil
+	if host:find("dtd.*%.") or host:find("mid%d%.") then
+		shortname=host:match("%.(.*)")
```

Comment:
Is the _shortname_ variable used somewhere else?  Or should it be **host=host:match("%.(.*)")** ?

Reply:
Good catch!
I looked at it so much I missed that! :facepalm: